app/queries/shortlist_foster_parent_query.rb

```diff
@@ -0,0 +1,34 @@
+class ShortlistFosterParentQuery
+  def initialize(placement_types: [])
+    @placement_types = placement_types
+  end
+
+  def call
```

Comment:
I note we are searching for a FosterParent, could we simplify this by searching for a PlacementSuitability ?

Reply:
I suppose we could, the result would be the same. Ultimately the desired outcome is a list of available Foster Parents though, so I think this is more reflecting the need. We could revert it and search on `PlacementSuitability` and then `include(:foster_parent)`.

The difference is that ideally these query objects should accept a relation to start with, I just didn't have time to implement it as I didn't need it. This relation may be chain-able and also probably ultimately that will be working on `FosterParent`. I hope that makes sense 🤔